src/modules/tests/vehicle-details/vehicle-details.cat-c.reducer.ts

```diff
@@ -0,0 +1,53 @@
+import * as vehicleDetailsActions from './vehicle-details.actions';
+// @TODO import Cat C Types
```

Comment:
Let's implement the TODOs now rather than later.

Reply:
As discussed, we'll address these as part of separate story.